coordinator-cli/src/signer.rs

```diff
@@ -147,18 +132,13 @@ impl<'a, 'b> Signer<'a, 'b> {
             // because often a big bunch of devices will register at similar times if they are daisy
             // chained together.
             loop {
+                std::thread::sleep(Duration::from_millis(100));
```

Comment:
What's the motivation behind this sleep?

Reply:
This loop hoses CPU otherwise. It should have a comment.